Packs/IBMResilientSystems/Integrations/IBMResilientSystems/IBMResilientSystems.py

```diff
@@ -622,7 +630,7 @@ def get_tasks_command(incident_id):
                 task_object['Form'] = task['form']
             if task['user_notes']:
                 task_object['UserNotes'] = task['user_notes']
-            task_object['Creator'] = task['creator']['fname'] + ' ' + task['creator']['lname']
+            task_object['Creator'] = task['creator_principal']['display_name']
```

Comment:
```suggestion
            task_object['Creator'] = task.get('creator_principal', {}).get('display_name')
```

Why was this changed? Was the API changed?